actions/user_actions.jsx

```diff
@@ -35,10 +35,10 @@ export function loadProfilesAndStatusesInChannel(channelId, page = 0, perPage =
     };
 }
 
-export function loadProfilesAndTeamMembers(page, perPage, teamId, options) {
+export function loadProfilesAndTeamMembers(page, perPage, teamId) {
     return async (doDispatch, doGetState) => {
         const newTeamId = teamId || getCurrentTeamId(doGetState());
-        const {data} = await doDispatch(UserActions.getProfilesInTeam(newTeamId, page, perPage, options));
+        const {data} = await doDispatch(UserActions.getProfilesInTeam(newTeamId, page, perPage));
```

Comment:
This redux action does accept an options field, we would just need to set sort to `""` https://github.com/mattermost/mattermost-redux/blob/master/src/actions/users.ts#L379

Reply:
Oh my god, I was running a month-old version of master in redux... Sorry!!   Why did we separate these into separate repositories? :)